lib/src/native/wrapper/sample_executor.cc

```diff
@@ -0,0 +1,111 @@
+// Derived from Chromium sample.
+
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
Native third party code should live in `third_party/`. (Dart third party code in `lib/src/third_party`.)

See https://opensource.google/docs/thirdparty/non-google3 .

Reply:
If this is just code for your own testing purposes, consider not committing it.

If you want to commit it, include a Dart script that somewhere that builds the code and runs it to see if it works. And make sure this script is executed on the continuous integration.